assets/scss/footer.scss

```diff
@@ -1,23 +1,21 @@
-.hale-page {
-  .govuk-footer .widget_social_widget {
-    //the styles here are just for the social media icon links.
-    font-size: 40px;
+.govuk-footer .widget_social_widget {
+  //the styles here are just for the social media icon links.
+  font-size: 40px;
```

Comment:
Could we add in rem as well, so we can do the browser-based font enlarge?

Reply:
Changed.